src/main/java/com/telefonica/iot/cygnus/sinks/OrionHDFSSink.java

```diff
@@ -90,7 +90,7 @@
     private String krb5LoginConfFile;
     private String krb5ConfFile;
     private boolean serviceAsNamespace;
-    private HDFSBackend persistenceBackend;
+    private HDFSBackendImpl persistenceBackend;
```

Comment:
Probably you are right, but it cannot be changed without disturbing the users, since there is a configuration parameter that has to be filled up with "OrionHDFSSink" string... We have to live with it!